### PR TITLE
I18n sync script

### PR DESCRIPTION
**Pull request by MarieVerdonck**
This PR adds a script to sync i18n message files to the latest english version. It will add missing keys to the other language files, and add TODO comments above them to indicate to the translator what has changed.

This will enable translators to start their work today, and keep tabs on how complete or incomplete the catalogs are.

To run the script type yarn run sync-i18n in the project root dir.

yarn run sync-i18n -h will give you an overview of the options

Please refer to https://wiki.duraspace.org/pages/viewpage.action?pageId=117735441 for further documentation.

### CODE REVIEW

File: scripts/sync-i18n-files.js [hunk 1]
```diff
@@ -0,0 +1,305 @@
...
+const NEW_MESSAGE_TODO = '// TODO New key - Add a translation';
+const MESSAGE_CHANGED_TODO = '// TODO Source message changed - Revise the translation';
+const COMMENTS_CHANGED_TODO = '// TODO Source comments changed - Revise the translation';
+
+const DEFAULT_SOURCE_FILE_LOCATION = 'resources/i18n/en.json5';
+const LANGUAGE_FILES_LOCATION = 'resources/i18n';
+
+parseCliInput();
+
+function parseCliInput() {
```
Comment (tdonohue) [inline scripts/sync-i18n-files.js]: Could you add a quick comment block here just to note what this function does?  I know it should be self explanatory based on the name, but it's always useful to comment larger functions.

File: scripts/sync-i18n-files.js [hunk 2]
```diff
@@ -0,0 +1,305 @@
...
+      console.error('Directory path of output file is not valid.');
+      console.log(program.outputHelp());
+      process.exit(1);
+    }
+
+    syncFileWithSource(program.targetFile, getOutputFileLocationIfExistsElseTargetFileLocation(program.targetFile));
+  }
+}
+
+function syncFileWithSource(pathToTargetFile, pathToOutputFile) {
```
Comment (tdonohue) [inline scripts/sync-i18n-files.js]: Same here, comments describing this function would be useful.  I really appreciate that you commented all the other larger functions below this one!  (As a sidenote, no need to comment the 1-2 line functions near the bottom of this file, those are fine as-is.)

**Comment by MarieVerdonck** [comment]
@tdonohue Missing documentation added and changed LF to CRLF if script run on windows machine.

**Comment by tdonohue** [comment]
@MarieVerdonck : I've re-reviewed & re-tested this and it now works perfectly on Windows.  Thanks for the updates!

If you could resolve the minor merge conflict (in `en.json5`), and get this PR updated (as needed), then I'll merge it immediately.  Thanks again.

**Comment by MarieVerdonck** [comment]
@tdonohue Newly added key in en.json5 merged and ran script again to update other language files

**Comment by tdonohue** [comment]
👍 Looks great now, thanks @MarieVerdonck for your hard work here.  Merging!

**Pull request merged**